tools/mason/MasonNew.chpl

```diff
@@ -47,18 +51,40 @@ proc masonNew(args) throws {
         else if arg == '--show' {
           show = true;
         }
+        else if arg.startsWith('--moduleName') {
+          isModuleName = true;
+          if arg.startsWith('--moduleName=') {
+            var res = arg.split("=");
+            moduleName = res[2];
+            name = args[2];
+            break;
+          }
+          else {
+            moduleName = args[i+2];
+            name = args[2];
+            break;
```

Comment:
If we are breaking, does that end arg parsing? What about arguments that come after `--moduleName foo`?